examples/led_animation_all_animations.py

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```

Comment:
@kattni Will do. I did a lot of the examples in an automated way since it would have probably taken ages not to so I'll correct that now. Thanks for catching that!